### PR TITLE
mirror: Fix typos and naming

### PR DESCRIPTION
**Pull request by dveeden**
### What problem does this PR solve? 

- `tiup mirrors` doesn't exist, but `tiup mirror` does
- Typo: `mirros` → `mirrors`
- Typo: `expirsAt` → `expiresAt`

### Check List 

Tests 

 - No code

Release notes:
```release-note
NONE
```

### CODE REVIEW

File: doc/user/mirrors.md [hunk 1]
```diff
@@ -68,16 +57,16 @@ If `--full` is specified, the official image will be cloned intact.
 ### 3. Platform limitation
 
 If you only want to clone packages for a particular platform, you can use `-os` and `-arch` to qualify:
-- `tiup mirros <target-dir> ---os=linux`
-- Just want to clone amd64 architecture: `tiup mirros <target-dir> --arch=amd64`
-- Just want to clone linux/amd64: `tiup mirros <target-dir> --os=linux --arch=amd64`
+- `tiup mirror <target-dir> ---os=linux`
```
Comment (AstroProfundis) [inline doc/user/mirrors.md:60]: I think it should be `tiup mirror clone <target-dir> ...`

File: doc/user/mirrors.md [hunk 2]
```diff
@@ -110,7 +99,7 @@ export TIUP_MIRRORS=/path/to/mirror
 tiup cluster xxx
 ```
 
-`/path/to/mirror` is the location of <target-dir> in `tiup mirrors <target-dir>`, or if in /tmp/package:
+`/path/to/mirror` is the location of <target-dir> in `tiup mirror <target-dir>`, or if in /tmp/package:
```
Comment (AstroProfundis) [inline doc/user/mirrors.md:102]: This should be `tiup mirror set <target-dir>` i think.

**Review by AstroProfundis — COMMENTED** [review]
Thanks a lot of the PR, we haven't updated the docs of mirror command for long time and many contents are not correct anymore.

File: doc/user/mirrors.md (hunk 1 above)
Comment (dveeden) [reply doc/user/mirrors.md:60]:
```suggestion
- `tiup mirror clone <target-dir> ---os=linux`
```

File: doc/user/mirrors.md (hunk 2 above)
Comment (dveeden) [reply doc/user/mirrors.md:102]:
```suggestion
`/path/to/mirror` is the location of <target-dir> in `tiup mirror set <target-dir>`, or if in /tmp/package:
```

**Comment by dveeden** [comment]
@AstroProfundis PTAL

File: doc/user/mirrors.md [hunk 3]
```diff
@@ -68,16 +57,16 @@ If `--full` is specified, the official image will be cloned intact.
...
 
 ### 4. Component version limited
 
 If you want to clone only one version of a component and not all versions, use `--<component>=<version>` to qualify, for example "
-- Just want to clone the v4 version of tidb: `tiup mirrors <target-dir> --tidb v4`
-- Just want to clone the v4 version of tidb, and all versions of tikv: `tiup mirros <target-dir> --tidb v4 --tikv all` 
-- Clone specific versions of all components that start a cluster: `tiup mirrors <target-dir> v4.0.0-rc`
+- Just want to clone the v4 version of tidb: `tiup mirror <target-dir> --tidb v4`
+- Just want to clone the v4 version of tidb, and all versions of tikv: `tiup mirror <target-dir> --tidb v4 --tikv all` 
+- Clone specific versions of all components that start a cluster: `tiup mirror <target-dir> v4.0.0-rc`
```
Comment (AstroProfundis) [inline doc/user/mirrors.md:69]: I think these examples should all be `tiup mirror clone xxx`?

Comment (dveeden) [reply doc/user/mirrors.md]: updated. PTAL

**Review by AstroProfundis — APPROVED** [review]

**Pull request merged**